torch_geometric/nn/conv/pan_conv.py

```diff
@@ -66,8 +66,12 @@ def forward(self, x: Tensor,
 
         adj_t: Optional[SparseTensor] = None
         if isinstance(edge_index, Tensor):
-            adj_t = SparseTensor(row=edge_index[1], col=edge_index[0],
-                                 sparse_sizes=(x.size(0), x.size(0)))
+            if is_torch_sparse_tensor(edge_index):
+                # TODO: handle PyTorch sparse tensor directly
+                adj_t = SparseTensor.from_torch_sparse_coo_tensor(edge_index)
```

Comment:
Can we assume the user will always pass a `coo` tensor?

Reply:
Instead of having an `if` statement here why not add a utitliy function 
```
from_torch_sparse_to_pyg_sparse(edge_index : torch.Tensor) -> SparseTensor:
"""
Converts a `edge_index` of type `torch.Tensor` to a `SparseTensor`. The `torch.Tensor`
can be sparse.
"""
```

Reply:
Sounds good. Will take it in another PR.

Reply:
Assuming we address this in another PR. I am good with these changes. 
Thanks !

Reply:
I think an `if` statement here is totally fine rather than introducing `to_pyg_sparse`. The goal is to get rid of `SparseTensor` in the long run, and so there is no real need for this method IMO.